internal/consensus/byzantine_test.go

```diff
@@ -327,69 +326,69 @@ func TestByzantineConflictingProposalsWithPartition(t *testing.T) {
 	// blocksSubs := make([]types.Subscription, n)
 	// reactors := make([]p2p.Reactor, n)
 	// for i := 0; i < n; i++ {
-	// 	// enable txs so we can create different proposals
-	// 	assertMempool(states[i].txNotifier).EnableTxsAvailable()
+	//	// enable txs so we can create different proposals
```

Comment:
I realize most of this is cosmetic changes to existing commented-out stuff, but could we get rid of the cmomented-out stuff? The test is already being skipped—if the issue is that the code doesn't even build, we are probably better off just deleting it. We can always recover the code from history.

Reply:
I'm inclined to do that in a different PR, but I'm definitely in support of this.